tests/conformance/pubsub/pubsub.go

```diff
@@ -386,11 +386,13 @@ func ConformanceTests(t *testing.T, props map[string]string, ps pubsub.PubSub, c
 			}
 
 			t.Logf("Calling Bulk Publish on component %s", config.ComponentName)
+			// Making use of entryMap defined above here to iterate through entryIds of messages published.
```

Comment:
adding comment here ... so that file is not missed in the deluge of certification go.mod changes :)